hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/DecommissionStatusSubCommand.java

```diff
@@ -134,9 +178,61 @@ private void printCounts(DatanodeDetails datanode, JsonNode counts, int numDecom
           return;
         }
       }
-      System.err.println("Error getting pipeline and container counts for " + datanode.getHostName());
-    } catch (NullPointerException ex) {
-      System.err.println("Error getting pipeline and container counts for " + datanode.getHostName());
+      System.err.println(getErrorMessage() + datanode.getHostName());
+    } catch (IOException e) {
+      System.err.println(getErrorMessage() + datanode.getHostName());
```

Comment:
Complete message can be formed once for a given datanode and can be used.
String errMsg = getErrorMessage() + datanode.getHostName();
```suggestion
      System.err.println(errMsg);
```